source/authentication/nsf-access.rst

```diff
@@ -18,6 +18,45 @@ Since `ACCESS`_ uses Open ID Connect (OIDC) you can see our :ref:`oidc documenta
 for more details on how to configure Open OnDemand with what CI Logon has provided in
 registering your application.
 
+Here's an example you can use to get started. Note that ``oidc_client_id`` and ``oidc_client_secret``
+are commented out because the are specific to your site.
```

Comment:
`the are specific to your site` to `they are specific to your site.`

Reply:
```suggestion
are commented out because they are specific to your site.
```

Reply:
Thanks! Do you know how to make suggestions like the one I did above? They make it easy to just commit the change directly.